### PR TITLE
Rename stack variables

### PR DESCRIPTION
**Pull request by robinkwilson**
Learnings recorded here : https://gist.github.com/netpro2k/4fc11d880a3b74bc630422e89108c19e

### CODE REVIEW

**Comment by robinkwilson** [comment]
- We'll need to publish the new template and market this out to people that couldn't connect before.

File: cloudformation/stack.yaml [hunk 1]
```diff
@@ -2474,36 +2474,28 @@ Resources:
...
         - IpProtocol: tcp
-          FromPort: !FindInMap [ServicesMeta, JanusExternal, Port]
-          ToPort: !FindInMap [ServicesMeta, JanusExternal, Port]
+          FromPort: !FindInMap [ServicesMeta, DialogExternalH, Port]
+          ToPort: !FindInMap [ServicesMeta, DialogExternalH, Port]
           CidrIp: !If [HasInboundCidrOverride, !Ref InboundCidrOverride, "0.0.0.0/0"]
         - IpProtocol: tcp
-          FromPort: !FindInMap [ServicesMeta, JanusTurnExternal, Port]
-          ToPort: !FindInMap [ServicesMeta, JanusTurnExternal, Port]
+          FromPort: !FindInMap [ServicesMeta, TurnExternalSH, Port]
```
Comment (robinkwilson) [inline cloudformation/stack.yaml:2485]: Dup begins from the above.

Comment (robinkwilson) [reply cloudformation/stack.yaml:2485]: missing chunk

**Comment by robinkwilson** [comment]
@vincentfretin Added comment to describe the variables:

```
    # Variables are suffixed with the instance types they apply to:
    # A = App, S = Streaming, H = Hybrid
```
A = used in app server (multi-server enterprise template) 
S = used in stream server (multi-server enterprise template)
H = hybrid stack (where both app + streaming services are combined on one box) used in HC Personal + 1 server stacks.

**Review by brianpeiris — APPROVED** [review]
LGTM

**Pull request merged**